xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -311,7 +311,9 @@ bool CDVDDemuxFFmpeg::Open(std::shared_ptr<CDVDInputStream> pInput, bool streami
       if (trySPDIFonly || (iformat && strcmp(iformat->name, "wav") == 0))
       {
         AVProbeData pd;
-        int probeBufferSize = 32768;
+        // use at least 32 KB probe buffer but be prepared so that an avio_read
+        // which might be done with max_packet_size does not write out of bounds
```

Comment:
Making the number of bytes read into probe buffer dependent on max_packet_size is wrong. i.e. set max_packet size to 8 and you'll end up with not enough data for probing. I prefer this patch: https://github.com/FernetMenta/kodi-agile/commit/d084f1f009bacea540720267a242e3a5e1fd508b

Reply:
Jep, that's better, way less complicated! Okay - when I pick it and replace mine?